zephyr/include/rtos/cache.h

```diff
@@ -8,12 +8,59 @@
 
 /* TODO: align with Zephyr generic cache API when ready */
 #define __SOF_LIB_CACHE_H__
+
+#ifndef CONFIG_IMX_ARM64
 #include <arch/lib/cache.h>
+#else
```

Comment:
This is not imx specific but arm64 specific. So, I wonder if we could only introduce CONFIG_ARM64 and somehow figure out that we are on imx arm64 by checking for CONFIG_IMX && CONFIG_ARM64?

Reply:
Fyi @nashif has a Zephyr Cache API in the works that will abstact the arch cache ops with a generic API..